README.md

```diff
@@ -27,6 +27,8 @@ Before you start developing your own integration, you might benefit from looking
 
 ```bash
 ./gradlew run
+or
```

Comment:
we should also explain here what the difference is between the two scenarios, it is not obvious without reading the Java code first.

Reply:
done